modernizer-annotation-processor/src/main/java/org/gaul/annotation_processsor/ModernizerAnnotationProcessor.java

```diff
@@ -53,43 +62,68 @@ public final boolean process(
         RoundEnvironment roundEnv
     ) {
         for (TypeElement annotation : annotations) {
-            List<String> annotatedClasses =
-                getAnnotatedClasses(roundEnv, annotation);
-            makeAnnotatedClassesFile(annotatedClasses);
+            List<List<String>> annotatedElements =
+                getAnnotatedElements(roundEnv, annotation);
+            makeAnnotatedElementsFiles(
+                annotatedElements.get(0), annotatedElements.get(1));
         }
         return true;
     }
 
-    private List<String> getAnnotatedClasses(
+    private List<List<String>> getAnnotatedElements(
```

Comment:
Instead of returning a `List<List<String>>`, could you make an object to represent the return type of this method?